app/gui/src/controller/graph/executed.rs

```diff
@@ -241,8 +241,8 @@ impl Handle {
     ) -> FallibleResult<Rc<MethodPointer>> {
         let registry = self.execution_ctx.computed_value_info_registry();
         let node_info = registry.get(&node).ok_or(NotEvaluatedYet(node))?;
-        let entry_id = *node_info.method_call.as_ref().ok_or(NoResolvedMethod(node))?;
-        self.project.suggestion_db().lookup_method_ptr(entry_id).map(Rc::new)
+        let method_pointer = node_info.method_call.as_ref().ok_or(NoResolvedMethod(node))?;
+        Ok(Rc::new(method_pointer.clone()))
```

Comment:
I'd like to avoid this clone: let's store the method pointer in the ComputedValueInfo under Rc.